gtest_parallel_tests.py

```diff
@@ -335,5 +337,135 @@ def root():
                                    'Test.case', 1))
 
 
+class TestFindTests(unittest.TestCase):
+  ONE_DISABLED_ONE_PASSED_TEST = {
+      "fake_unittests": {
+          "FakeTest": {
+              "Test1": None,
+              "DISABLED_Test2": None,
+          }
+      }
+  }
+  ONE_FAILED_ONE_PASSED_TEST = {
+      "fake_unittests": {
+          "FakeTest": {
+              # Failed (and new) tests have no recorded runtime.
+              "FailedTest": None,
+              "Test": 1,
+          }
+      }
+  }
+  ONE_TEST = {
+      "fake_unittests": {
+          "FakeTest": {
+              "TestSomething": None,
+          }
+      }
+  }
+  COMPLEX_TEST = {
```

Comment:
Can you put something in here that describes what you want the test to contain (rather than just being complex), such as MULTIPLE_BINARIES_MULTIPLE_TESTS_ONE_FAILURE or whatever? Even if the name gets sucky-long, it's better than a non-descript name.